src/components/donation/Steps/index.tsx

```diff
@@ -17,25 +14,16 @@ export type ConfigParams = {
 type Props = { className?: string } & ConfigParams;
 
 export function Steps({ className = "", ...params }: Props) {
-  const { showModal } = useModalContext();
   const state = useGetter((state) => state.donation);
-  const handleOpenKado = useCallback(
-    () => showModal(KadoModal, {}),
```

Comment:
@abbasali1994, @SovereignAndrey  - is `<KadoModal/>` not used anymore ? perhaps we could delete it .

Reply:
Its unclear from ticket are we still providing or plan to provide ways to but crypto from webapp. @SovereignAndrey your inputs?

Reply:
We can leave it for now, since we might want to have a "buy crypto" link with that Kado modal elsewhere.